scripts/kevm-integration.sh

```diff
@@ -33,7 +29,6 @@ make clean
 git submodule update --init --recursive
```

Comment:
Probably don't want to either commit the submodule, run `make clean` or `git submodule update --init --recursive`. It will wipe out the `.build/k` directory and re-clone KEVM's version.